ibis/tests/all/test_aggregation.py

```diff
@@ -146,6 +151,7 @@ def test_aggregation(
                 .reset_index()
             ),
             id='group_concat',
+            marks=pytest.mark.skip_backends((MSSQL,)),
```

Comment:
any reason to use `skip_backend` here instead of `xfail_backends`?

Reply:
no reason at all. changed to xfail_backends